services/web/server/src/simcore_service_webserver/exporter/formatters/formatter_v1.py

```diff
@@ -82,22 +82,22 @@ async def extract_download_links(
 
     log.debug("s3 files metadata %s: ", s3_metadata)
 
```

Comment:
MINOR: At this level, why not to refer to them as ``datcore``? At the end of the day the datcore-adapter is an interface to datacore resources and hides pennsieve details, right?

Reply:
ok. I actually replaced blackfynn with pennsieve in a find/replace.
If you talk about the comment on line 85 ok I can change it.
about the exception, if it goes all the way to the frontend, I would talk about what the user knows. and in that case I think pennsieve makes more sense.